rfc/graphql-schema/additionalTypeDefs/byPage/userManagement.ts

```diff
@@ -48,6 +48,12 @@ export default /* GraphQL */ `
       SomeChangeCustomerRoleError
     | AnotherChangeCustomerRoleError
 
+  input ChangeCustomerRoleInput {
+    companyId: ID!
+    customerId: Int!
+    roleId: ID!
```

Comment:
🤔 if this involved more than one property, I'd argue we should scope it into a `data` property, but since it is just one, I'm a bit conflicted with the extra "complexity"